packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -62,12 +64,12 @@ function SquadPostContent({
   };
 
   useEffect(() => {
-    if (!post?.id) {
+    if (!post?.id || !user?.id) {
       return;
     }
 
     onSendViewPost(post.id);
-  }, [post?.id, onSendViewPost]);
+  }, [post.id, onSendViewPost, user?.id]);
```

Comment:
@idoshamun I just wanted to double check with you if this had any potential impact on the metrics/feed for anonymous user?

Reply:
This could have had minor impact in case of a race condition. But I think it's an edge case, so I wouldn't worry about that for now. Thanks for pointing this out!